src/utils/lineGraph.go

```diff
@@ -66,7 +66,11 @@ func (l *LineGraph) Draw(buf *ui.Buffer) {
 	// sort the series so that overlapping data will overlap the same way each time
 	seriesList := make([]string, len(l.Data))
 	i := 0
-	l.MaxVal = 1
+
+	if l.MaxVal == 0 {
+		l.MaxVal = 1
+	}
```

Comment:
What does this do?

Reply:
Ahh. So this used when we're adjusting the MaxVal according to the data given sent to it. In the case where the maximum value in the given data is 0, we don't want MaxVal to be zero. So this condition ensures it's always greater than 0. Otherwise there's gonna be seg faults while rendering the graph